RELEASING.md

```diff
@@ -139,23 +139,27 @@ Here are the steps:
-12. [Mac only] Once the binaries are available on the website you need to run the [homebrew cask updater](https://github.com/AdoptOpenJDK/homebrew-openjdk/wiki/Running-the-cask-updater) which will create a pull request [here](https://github.com/AdoptOpenJDK/homebrew-openjdk/pulls). Normally George approves these but in principle as long as the CI passes, they should be good to approve. You don't need to wait around and merge the PR's because the Mergify bot will automatically do this for you as long as somebody has approved it.
-13. Publicise the Temurin release via slack on Adoptium #release
-14. If desired, find someone with the appropriate authority (George, Martijn, Shelley, Stewart) to post a tweet about the new release from the Adoptium twitter account
+1. Once the job completes successfully, check the binaries have uploaded to GitHub at somewhere like https://github.com/adoptium/temurin8-binaries/releases/tag/jdk8u302-b08
+1. Within 15 minutes the binaries should be available on the website too at e.g. https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot (NOTE: If it doesn't show up, check whether the API is returning the right thing (e.g. with a link such as [this](https://api.adoptium.net/v3/assets/feature_releases/17/ga?architecture=x64&heap_size=normal&image_type=jre&jvm_impl=hotspot&os=linux&page=0&page_size=10&project=jdk&sort_method=DEFAULT&sort_order=DESC&vendor=eclipse), and that the `.json` metadata files are uploaded correctly)
+1. Since you have 15 minutes free, use that time to update https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/support.handlebars which is the source of  https://adoptium.net/support.html and (if required) the supported platforms table at https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/supported_platforms.handlebars which is the source of https://adoptium.net/supported_platforms.html, and also update https://adoptium.net/release_notes.html ([Sample change](https://github.com/adoptium/adoptium.net/pull/675/commits/563d8e2f0d9d26500a7e8d9eca61b491f73f1f37)
+1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/pull/113581). If in doubt reach out to George (gdams) as he's a maintainer.
```

Comment:
Clarify whether it needs to be three separate PRs against the three casks.
Also possibly provide a link to a JDK8 one alongside the JDK17 one as it's a "simpler" case without the change of `U` (which is also good as an example here of course!)

Reply:
Perhaps add a note that saying that you can easily copy the the SHAs for inclusion in the PR from the temurin download website for [8](https://adoptium.net/releases.html?variant=openjdk8&jvmVariant=hotspot), [11](https://adoptium.net/releases.html?variant=openjdk11&jvmVariant=hotspot) and [17](https://adoptium.net/releases.html?variant=openjdk17&jvmVariant=hotspot)